modules/luci-mod-admin-full/luasrc/model/cbi/admin_network/wifi_add.lua

```diff
@@ -44,7 +44,7 @@ m.hidden = {
 
 if iw and iw.mbssid_support then
 	replace = m:field(Flag, "replace", translate("Replace wireless configuration"),
-		translate("An additional network will be created if you leave this checked."))
+		translate("An additional network will be created if you leave this unchecked."))
```

Comment:
You should also update the translation sources -- not sure if there's a way to determine if translations are accurate.

See: https://github.com/openwrt/luci/search?utf8=%E2%9C%93&q=An+additional+network+will+be+created